pkg/client/observatorium/api.go

```diff
@@ -143,6 +143,14 @@ func (obs *ServiceObservatorium) GetMetrics(metrics *KafkaMetrics, namespace str
 			`kafka_topic:kafka_log_log_size:sum`,
 			fmt.Sprintf(`%s, namespace=~'%s'`, privateTopicFilter, namespace),
 		},
+		{
+			`kas_topic_partition_log_size_bytes`,
+			fmt.Sprintf(`%s, namespace=~'%s'`, privateTopicFilter, namespace),
+		},
+		{
+			`kas_broker_partition_log_size_bytes_top50`,
+			fmt.Sprintf(`namespace=~'%s'`, namespace),
```

Comment:
why no private topic filter for this metric?

Reply:
Because it is already filtered on the recording rule: https://github.com/bf2fc6cc711aee1a0c2a/observability-resources-mk/pull/246/files#diff-404a5eb3a8383bdb003bd42a6b7bfb983b6921cede7804390a92c09707d226c0R44

Reply:
:+1: